docs/how-to-maintain.rst

```diff
@@ -0,0 +1,97 @@
+How To Maintain
+===============
+
+This document is dedicated to maintainers of GETTSIM.
+
+
+Versioning
+----------
+
+GETTSIM adheres in large parts to `semantic versioning <https://semver.org>`_. Thus, for
+a given version number ``major.minor.patch``
+
+* ``major`` is incremented when you make incompatible API changes.
+* ``minor`` is incremented when you add functionality which is backwards compatible.
+* ``patch`` is incremented when you make backwards compatible bug fixes.
+
+Branching Model
+---------------
+
+The branching model for GETTSIM is very simple.
+
+1. The master branch is the target of ongoing development. Every PR should be merged
+   into master.
+
+   (The only deviation can happen with respect to backports. See :ref:`backports` for
```

Comment:
This sentence can go. Or we need to say that this may happen additionally. This sounds too much like an either / or thing.

Reply:
done